lib/index.js

```diff
@@ -48,10 +48,12 @@ const express = require('express'),
   compareVersions = require('compare-versions')
 
 function Server (opts) {
+  const FILEUPLOADSIZELIMIT = process.env.FILEUPLOADSIZELIMIT || '10mb'
```

Comment:
Could you please add a note about this to README.md?

Reply:
done